src/coreclr/CMakeLists.txt

```diff
@@ -112,7 +112,7 @@ endif(WIN32)
 # Configure compiler settings for environment
 #----------------------------------------------------
 include(configurecompiler.cmake)
-include(configuretools.cmake)
+include(${CLR_COMMON_CROSS_DIR}/configuretools.cmake)
```

Comment:
Can you please put the files into `eng/common` instead of `eng/common/cross` and rename the variable to not to contain the "CROSS" in the name? They don't have anything to do with cross target building.

Reply:
Renamed to `CLR_COMMON_DIR`.